.changeset/hip-melons-clean.md

```diff
@@ -0,0 +1,7 @@
+---
+'xstate': major
+---
+
+Persisted state will no longer contain actions, since they have already been executed. Actions will not be replayed.
```

Comment:
This is no longer true since you changed the implementation from discarding those actions when persisting to ignoring them when rehydrating. Perhaps for the time being we should do both or adjust this sentence? 

I'll explore tomorrow how feasible it is to remove `State['actions']`

Reply:
👋

Reply:
Adjusted